torch/testing/_internal/common_methods_invocations.py

```diff
@@ -681,6 +681,47 @@ def sample_inputs_index_select(op_info, device, dtype, requires_grad):
                         0, torch.tensor(0, dtype=torch.int64, device=device))),
             )
 
+def sample_inputs_sort(op_info, device, dtype, requires_grad):
+    def apply_grad(t):
+        if dtype in floating_types_and(torch.float16, torch.bfloat16):
+            t.requires_grad_(requires_grad)
+
+    def small_3d_unique(dtype, device):
+        res = torch.randperm(S * S * S, dtype=torch.int64, device=device).view(S, S, S)
+        res = res.to(dtype)
+        apply_grad(res)
+        return res
+
+    samples = []
+
+    # Test cases for small 3d tensors.
+    # Imitates legacy tests from test/test_torch.py
+    self = small_3d_unique(dtype, device)
+    dims = range(-3, 3)
+    flag = [True, False]
+    for dim, descending, stable in product(dims, flag, flag):
+        # default schema without stable sort
+        samples.append(SampleInput((self, dim, descending)))
+        # schema with stable sort, no CUDA support yet
+        if not device.startswith('cuda'):
+            samples.append(
+                SampleInput(self, kwargs=dict(dim=dim, descending=descending, stable=stable))
+            )
+
+    # Test cases for scalar tensor
+    scalar = (torch.randn(1, device=device) * 100).to(dtype)
```

Comment:
I would just create the scalar tensor directly: `torch.tensor(1, device=device, dtype=dtype)` or whatever number you like.